resources/contributors.md

```diff
@@ -0,0 +1,49 @@
+---
+id: contributors
+title: Our Contributors
+sidebar_label: Contributors
+slug: /contributors
+---
+
+Thank you for contributing to the UWindsor Computer Science Society Wiki! 
+
+#### **Ryan Prairie**
++ https://github.com/prairir
++ prairie@uwindsor.ca
+
+#### **Aman Patel**
++ https://github.com/amanpatel123
+
+#### **Jeremie Bornais**
++ https://jeremie.bornais.ca/
++ https://github.com/jere-mie
++ borna113@uwindsor.ca
+
+#### **Abdul Arif**
++ https://abdularif.tech/
++ https://github.com/abdularif0705
++ arifb@uwindsor.ca
+
+#### **Isaac Kilbourne**
++ https://kilbouri.github.io/
++ https://github.com/kilbouri
++ kilbouri@uwindsor.ca
+
+#### **Christine Wong**
++ https://github.com/Lillliant
++ wong93@uwindsor.ca
+
+#### **Ahmad Munim**
++ https://github.com/ahmadmunim
++ munima@uwindsor.ca
+
+#### **Arunesh Choudhary**
++ https://github.com/choudh91
+
+#### **Hasan Ahmad**
```

Comment:
If you could update my name with "Hassan" (double s)

Reply:
Fixed.